### PR TITLE
Added dynamic cast fetching in watch screen + improved css

### LINKED ISSUE #19: Styling: Bigger cast details. Position fixed

### LINKED ISSUE #20: Fetch the cast details dynamically

### LINKED ISSUE #64: Styling Changes for the application

### PR DESCRIPTION
**Pull request by vickymhs**
1. Added dynamic cast fetching in watch screen 
2. improved css.

@rahulkalita8 @isha-bansal0115 @ArzaEshita @lgdeloss @kirantejatummuri 

Please review this

Screen Shot 2021-09-26 at 4 52 31 PM

### CODE REVIEW

File: client/js/netflix/watchScreen.js [hunk 1]
```diff
@@ -111,11 +103,13 @@ var mutationObserver = new MutationObserver(function(mutations) {
             const title = getTitle();
             
             // Get the cast details
-            // const movieDetails = getCastDetails(title);
-            const casts = getCastDetails1(title);
-            console.log("printing cast details ", casts);
-            console.log(typeof(casts));
-            addToWatchScreen(casts)
+            const response = getCastDetails(title).then(result => {
```
Comment (isha-bansal0115) [inline client/js/netflix/watchScreen.js:106]: if you want, you can use async await. This also works though

File: client/js/netflix/watchScreen.js [hunk 2]
```diff
@@ -111,11 +103,13 @@ var mutationObserver = new MutationObserver(function(mutations) {
...
+                    var castString = result.cast;
```
Comment (isha-bansal0115) [inline client/js/netflix/watchScreen.js]: shall we check for null here?

let castString = result ? result.cast : null;

and use let or const if possible.

and then below you can do

const casts = castString && castString !== 'N/A' ? castString.split(',') : 'Cast Information not available';
addToWatchScreen(casts)

**Review by isha-bansal0115 — CHANGES_REQUESTED** [review]
Minor changes suggested

File: client/js/netflix/watchScreen.js (hunk 1 above)
Comment (vickymhs) [reply client/js/netflix/watchScreen.js:106]: I tried that.. Somehow I couldn't seamlessly replicate that. So I did it with promises. I hope it should also do the intended behavior. If I get some more time, I will check on that again

File: client/js/netflix/watchScreen.js (hunk 2 above)
Comment (vickymhs) [reply client/js/netflix/watchScreen.js]: Yeah makes sense! I will do that

**Review by isha-bansal0115 — APPROVED** [review]
we can further improve but approving for now

**Pull request merged**